src/test/java/api/item/ItemLastCheckInTests.java

```diff
@@ -0,0 +1,132 @@
+    JsonObject lastCheckIn = itemsClient.get(item.getId())
+      .getJson()
+      .getJsonObject("lastCheckIn");
+
+    assertThat(lastCheckIn.getString("dateTime"), is(checkInDate.toString()));
+    assertThat(lastCheckIn.getString("servicePointId"), is(servicePointId.toString()));
+    assertThat(lastCheckIn.getString("staffMemberId"), is(nullValue()));
+
+    APITestContext.setUserId("");
+    loansFixture.checkInByBarcode(item, checkInDate, servicePointId);
+    lastCheckIn = itemsClient.get(item.getId())
+      .getJson()
+      .getJsonObject("lastCheckIn");
+
+    assertThat(lastCheckIn.getString("dateTime"), is(checkInDate.toString()));
+    assertThat(lastCheckIn.getString("servicePointId"), is(servicePointId.toString()));
+    assertThat(lastCheckIn.getString("staffMemberId"), is(nullValue()));
+  }
+
+  @Test
+  public void shouldBeAbleToCheckinItemWithoutLoan()
+    throws InterruptedException, MalformedURLException, TimeoutException,
+    ExecutionException {
+
+    IndividualResource item = itemsFixture.basedUponSmallAngryPlanet();
+    UUID servicePointId = servicePointsFixture.cd1().getId();
+    DateTime checkInDate = DateTime.now();
+
+    loansFixture.checkInByBarcode(item, checkInDate, servicePointId);
+    JsonObject lastCheckIn = itemsClient.get(item.getId())
+      .getJson()
+      .getJsonObject("lastCheckIn");
+
+    assertThat(lastCheckIn.getString("dateTime"), is(checkInDate.toString()));
+    assertThat(lastCheckIn.getString("servicePointId"), is(servicePointId.toString()));
+    assertThat(lastCheckIn.getString("staffMemberId"), is(APITestContext.getUserId()));
+  }
+
+  @Test
+  public void shouldBeAbleCheckinItemWithoutLoanMultipleTimes()
+    throws InterruptedException, MalformedURLException, TimeoutException,
+    ExecutionException {
+
+    IndividualResource item = itemsFixture.basedUponSmallAngryPlanet();
+    UUID servicePointId = servicePointsFixture.cd1().getId();
+    DateTime firstCheckInDate = DateTime.now();
+
+    loansFixture.checkInByBarcode(item, firstCheckInDate, servicePointId);
+    JsonObject lastCheckIn = itemsClient.get(item.getId())
+      .getJson()
+      .getJsonObject("lastCheckIn");
+
+    assertThat(lastCheckIn.getString("dateTime"), is(firstCheckInDate.toString()));
+    assertThat(lastCheckIn.getString("servicePointId"), is(servicePointId.toString()));
+    assertThat(lastCheckIn.getString("staffMemberId"), is(APITestContext.getUserId()));
+
+    loansFixture.checkInByBarcode(item, DateTime.now(), servicePointId);
+
+    UUID servicePointId2 = servicePointsFixture.cd2().getId();
+    APITestContext.setUserId(UUID.randomUUID().toString());
+    loansFixture.checkInByBarcode(item, DateTime.now(), servicePointId2);
+    lastCheckIn = itemsClient.get(item.getId())
+      .getJson()
+      .getJsonObject("lastCheckIn");
+
+    assert (DateTime.parse(lastCheckIn.getString("dateTime"))
+      .isAfter(firstCheckInDate));
```

Comment:
Alternatively, a second explicit check in date could be defined, rather than relying on the less strict after check.

Reply:
Done.